src/modules/flight_mode_manager/tasks/ManualAcceleration/FlightTaskManualAcceleration.cpp

```diff
@@ -59,6 +59,7 @@ bool FlightTaskManualAcceleration::activate(const trajectory_setpoint_s &last_se
 
 bool FlightTaskManualAcceleration::update()
 {
+	setMaxDistanceToGround(_sub_vehicle_local_position.get().hagl_max_xy);
```

Comment:
what about `hagl_max_z` ?

Reply:
would be used in the altitude controller when the max distance was not set before.